plutus-core/changelog.d/20240517_191403_effectfully_polish_evaluation_errors.md

```diff
@@ -0,0 +1,3 @@
+### Removed
+
+- `unsafeRunCekNoEmit` and all `unsafeEvaluate*` functions in #6043. To replace e.g. `unsafeEvaluateCek` you can use `evaluateCek` in combination with `unsafeToEvaluationResult`.
```

Comment:
Also Changed section , renamed unsafeExtractEvaluationResult   to unsafeToEvaluationResult